homeassistant/components/bang_olufsen/media_player.py

```diff
@@ -358,6 +462,135 @@ def _async_update_volume(self, data: VolumeState) -> None:
 
         self.async_write_ha_state()
 
+    @callback
+    async def _async_update_name_and_beolink(self) -> None:
+        """Update the device friendly name."""
```

Comment:
Should this happen in a separate PR, or is it somehow directly related to the grouping?

Reply:
This is to show the correct friendly name in the extra state attribute `beolink` -> `self`